NameLayerMain/src/vg/civcraft/mc/namelayer/listeners/MercuryMessageListener.java

```diff
@@ -31,42 +34,58 @@ public MercuryMessageListener() {
 	public void onMercuryMessage(AsyncPluginBroadcastMessageEvent event){
 		if (!event.getChannel().equalsIgnoreCase("namelayer"))
 			return;
-		String[] message = event.getMessage().split(" ");
+		String[] message = event.getMessage().split("|");
```

Comment:
There already exist groups with pipes though as far as we know. You would need a database version upgrade as well to modify any existing groups with pipes in their names.